Quadriga/src/main/java/edu/asu/spring/quadriga/service/resolver/impl/ProjectHandleResolverManager.java

```diff
@@ -43,32 +44,59 @@ public boolean saveProjectHandleResolver(IProjectHandleResolver resolver, String
             resolver.setId(id);
             resolver.setUsername(username);
         }
-        
+
         ProjectHandleResolverDTO resolverDto = mapper.mapProjectHandleResolver(resolver);
         resolverDao.saveOrUpdateDTO(resolverDto);
-        
+
         return true;
     }
-    
-    /* (non-Javadoc)
-     * @see edu.asu.spring.quadriga.service.resolver.impl.IProjectHandleResolverManager#getProjectHandleResolver(java.lang.String)
+
+    /*
+     * (non-Javadoc)
+     * 
+     * @see edu.asu.spring.quadriga.service.resolver.impl.
+     * IProjectHandleResolverManager#getProjectHandleResolver(java.lang.String)
      */
     @Override
     public IProjectHandleResolver getProjectHandleResolver(String id) {
         ProjectHandleResolverDTO dto = resolverDao.getDTO(id);
         return mapper.mapProjectHandleResolver(dto);
     }
-    
-    /* (non-Javadoc)
-     * @see edu.asu.spring.quadriga.service.resolver.impl.IProjectHandleResolverManager#getProjectHandleResolvers(java.lang.String)
+
+    /*
+     * (non-Javadoc)
+     * 
+     * @see edu.asu.spring.quadriga.service.resolver.impl.
+     * IProjectHandleResolverManager#getProjectHandleResolvers(java.lang.String)
      */
     @Override
     public List<IProjectHandleResolver> getProjectHandleResolvers(String username) {
         List<ProjectHandleResolverDTO> resolverDtos = resolverDao.getProjectResolversForUser(username);
-        
+
         List<IProjectHandleResolver> resolvers = new ArrayList<IProjectHandleResolver>();
         resolverDtos.forEach(dto -> resolvers.add(mapper.mapProjectHandleResolver(dto)));
-        
+
         return resolvers;
     }
+
+    /**
+     * deletes the resolver object from the database
```

Comment:
Sentences start with an upper case letter. Also, this Javadoc should say more concretely what's happening. That it deletes a resolver is expected from the name. But the business logic it is doing is important.